### ISSUE TITLE
2.1.5 Broken 

### ISSUE THREAD

**Issue by frodrigo** [maplibre/maplibre-gl-js#997]
While updating from 2.1.1 to 2.1.5.

On a vuejs/nuxt with type script. 2.1.1 was fully building, while 2.1.5 end up with

```
Error [ERR_REQUIRE_ESM]: require() of ES Module /home/fred/perso/teritorio/vido/node_modules/maplibre-gl/dist/maplibre-gl.js from /home/fred/perso/teritorio/vido/node_modules/vue-server-renderer/build.prod.js not supported.
maplibre-gl.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead rename maplibre-gl.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in /home/fred/perso/teritorio/vido/node_modules/maplibre-gl/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).

    at /home/fred/perso/teritorio/vido/node_modules/vue-server-renderer/build.prod.js:1:77600
    at Object.<anonymous> (webpack:/external "maplibre-gl":1:0)
    at __webpack_require__ (webpack/bootstrap:25:0)
    at Module.<anonymous> (pages/index.js:6108:29)
    at __webpack_require__ (webpack/bootstrap:25:0)
    at async server.js:1530:21
    at async Promise.all (index 0)
    at async getRouteData (node_modules/.cache/nuxt/utils.js:181:0)
    at async Promise.all (index 0)
    at async setContext (node_modules/.cache/nuxt/utils.js:259:0)
    at async createApp (node_modules/.cache/nuxt/index.js:158:0)
    at async module.exports.__webpack_exports__.default (node_modules/.cache/nuxt/server.js:85:0)
```

I a know, `required()` is not used in the project.

node: v17.3.0
yarn: 2.4.1

**Comment by HarelM** [comment]
Uff... Seems like package.json file is missing in the dist folder again...

**Comment by wipfli** [comment]
Jup https://unpkg.com/browse/maplibre-gl@2.1.5/dist/

**Comment by wipfli** [comment]
I will file a pull request. Thanks for reporting.

**Comment by HarelM** [comment]
It should be part of the release.yml as well I guess. It was in the post_ts_something which we removed and didn't fully understood there was another step there...

**Comment by birkskyum** [comment]
#928

**Comment by wipfli** [comment]
Should be fixed in [v2.1.6-pre.1](https://github.com/maplibre/maplibre-gl-js/releases/tag/v2.1.6-pre.1). Can you give it a try @frodrigo ?

**Comment by frodrigo** [comment]
> Should be fixed in [v2.1.6-pre.1](https://github.com/maplibre/maplibre-gl-js/releases/tag/v2.1.6-pre.1). Can you give it a try @frodrigo ?

Yes it is ok on my side now ! Thank you.

**Comment by wipfli** [comment]
Cool
